spring-boot/admin/src/main/resources/messages.properties

```diff
@@ -75,7 +75,7 @@ admin.proxies.list.title=Manage Service Gateways
 admin.proxies.list.col.name=Name
 admin.proxies.list.col.description=Description
 admin.proxies.list.col.identifier = Gateway Identifier
-admin.proxies.list.col.services=Provided Service APIs
+admin.proxies.list.col.services=Provided Service API / Identifier (URL)
```

Comment:
To keep it consistent with other places, I would put the identifier also in brackets in this case (not behind a "/").

Maybe use `Provided Service API (Identifier - URL)`?